chapters/revisions.tex

```diff
@@ -178,7 +178,7 @@ \subsection{Main changes in Modelica 3.5}\label{main-changes-in-modelica-3-5}
 Ticket \href{https://github.com/modelica/ModelicaSpecification/issues/2288}{\#2288}.
 \item Added test-case annotation for incorrect models, \cref{modelica:TestCase}.
 Ticket \href{https://github.com/modelica/ModelicaSpecification/issues/2340}{\#2340}.
-\item Clarified choicesAllMatching, \cref{annotation-choices-for-suggested-redeclarations-and-modifications}.
+\item Clarified \lstinline!choicesAllMatching!, \cref{annotations-for-suggested-redeclarations-and-modifications}.
```

Comment:
Never making edits to _revisions.tex_ isn't possible with a document that gets reorganized over time.  I also don't see the value of un-touched LaTeX source code.

As an alternative to updating cross references from the revision history, one could consider replacing obsolete cross references with a standard formulation such as _(original context of change is no longer available)_, instead of updating the cross references to point to the current best available approximation of the original reference.